webapp/src/index.js

```diff
@@ -3,8 +3,6 @@
 
 import React from 'react';
 
-import {getConfig} from 'mattermost-redux/selectors/entities/general';
-
```

Comment:
Can we add this line back into this file? That should be the last change needed to merge the PR

Reply:
sorry about the force push, had an issue with a commit that shouldn't have gone there. Will keep that in mind.

Reply:
Thank you 🙂

btw having extra/unwanted commits in the PR is fine, because the squash merge will combine it into one commit in the master commit history